core/domain/exp_services.py

```diff
@@ -1320,7 +1724,16 @@ def _is_suggestion_valid(thread_id, exploration_id):
 
 
 def _is_suggestion_handled(thread_id, exploration_id):
-    """Checks if the current suggestion has already been accepted/rejected."""
+    """Checks if the current suggestion has already been accepted/rejected.
+
+    Args:
+        thread_id: str. ID of the object of FeedbackThread model.
+        exploration_id: str. ID of the exploration domain object.
+
+    Returns:
+        bool. True if the current suggestion is already beeb accepted or 
```

Comment:
Whether the current suggestion has already been acted upon (i.e., accepted or rejected).